content/documentation/github.md

```diff
@@ -1,101 +1,143 @@
 
-With a project on GitHub, it’s possible for others to make changes and commits to it as well. There are three basic ways this can happen:
 
-- They can fork (copy) your repository, make changes to it and then submit a pull request to you to review and merge the changes into your project. This can be more complicated to manage, but it’s safer as it ensures they can’t unintentionally do anything irreversible to your project.
-- You can make them a collaborator on your repository in which case they can work on it directly just as you would. This is generally easier to manage, but grants them full access to your repository.
-- If your project is hosted in a GitHub Organization account, you can add them to the organization team.
+## Manage Project Changes on GitHub
 
-When collaborating with others, be communicative about what areas of the project you’re working on and try to avoid cross overs. If you have multiple people connected to a project and pushing commits to the GitHub repository, there’s a good chance you will at some point make changes that are incompatible with someone else’s changes.The trick with multiple people working in a single project is managing these potential change conflicts. Git is designed to deal with these sorts of issues.
+There are three basic steps for making and managing changes to your project on GitHub:
 
-When working with others on a project, use branches as a way of managing different users' changes and avoiding conflicts. Users can make changes on their branch before submitting a pull request to have the changes committed to the master branch. This allows you to review these changes and identify possible conflicts before they are added to your project. There are a number of guides for forking repositories, using branches, and making pull requests online.
+1. Save changes in your text editor
+2. Commit those changes to the git repository
+3. Push those commits up to GitHub
 
-## Installing an Existing Quire Project from GitHub
+### Save Changes
 
-1. Open GitHub Desktop and go to File > Clone Repository (Shift+Cmd+O).
+Open your project in your text editor (File > Open > [your project name]), make, and save changes. When you go into GitHub Desktop, you should see a list of all the files added, deleted, or edited in the left-hand pane. On the right, you'll see exactly what changes were made inside each file. Old text will be highlighted in red, while any new text will be highlighted in green. This enables you to compare and review changes easily.
 
-2. Click the URL tab, and enter the URL or GitHub username and repository name where indicated.
+{{< q-class "box tip" >}}
+- Save changes with either Command-S or File > Save. If working on multiple markdown files, make sure to save each file individually. You will see a little blue dot at the top of the page if there are any unsaved changes.   
```

Comment:
Not sure if we need to be explicit all the time, but on Windows it's Control-S, rather than Command-S.

Reply:
I've been thinking about this a lot and I think we should either include both or none. I might need to go back through docs and add PC shortcuts in. Don't want us to have a bias.